model_compression_toolkit/keras/back2framework/model_builder.py

```diff
@@ -312,6 +315,13 @@ def _quantize_multiple_nbits(layer):
         # clone each layer in the model and apply _quantize to the layer.
         model = tf.keras.models.clone_model(model, input_tensors=None, clone_function=_quantize_multiple_nbits)
 
+        # add configurable input layers in case of activation mixed-precision
+        model_inputs = graph.get_inputs()
+        for inp in model_inputs:
+            if inp.is_activation_quantization_enabled() and not inp.is_all_activation_candidates_equal():
```

Comment:
Add comments about what is a model transformer, and the list of transformations we pass, yada yada yada...